init_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/fnproject/cli/langs"
+	"github.com/urfave/cli"
+	yaml "gopkg.in/yaml.v2"
+)
+
+func TestInit(t *testing.T) {
+
+	testname := "test-init"
+	testdir, err := ioutil.TempDir("", testname)
+	if err != nil {
+		t.Fatalf("ERROR: Failed to make tmp test directory: err: %v", err)
```

Comment:
better to use `err.String()`, not sure that `%v` would give human-readable err representation.

Reply:
It does.